### ISSUE TITLE
`bun add   .`   cannot find `package.json` 

### ISSUE THREAD

**Issue by souhailaS** [oven-sh/bun#6322]
### What version of Bun is running?

1.0.3

### What platform is your computer?

x64

### What steps can reproduce the bug?

run `bun add  .` inside  folder that has a package.json file 

### What is the expected behavior?

To install the current module just like if i run `npm install  .` 

### What do you see instead?

```
error: MissingPackageJSON
Bun could not find a package.json file.
```

**Comment by sangeethkc** [comment]
I just tried this and I have the same issue.
![2023-10-06_11-34](https://github.com/oven-sh/bun/assets/43245950/e37fa4d7-d81e-4dc6-9d58-fd79f037f4ab)

**Comment by dylan-conway** [comment]
This is fixed in latest bun `v1.1.13`, closing as complete. Please reopen or create a new issue if you continue to have issues with `bun install`.
